lib/ofx_preproc.cpp

```diff
@@ -308,7 +308,11 @@ int ofx_proc_file(LibofxContextPtr ctx, const char * p_filename)
             const char* inchar = s_buffer.c_str();
             char * outchar = iconv_buffer;
             int iconv_retval = iconv (conversion_descriptor,
+#ifdef _WIN32 //Win32's iconv implementation takes a const char** inchar
```

Comment:
I agree with the change here in general.

BUT: If you enclose some code in an #ifdef-#else-#endif, then this clause be limited to ONLY the code that actually changes with this condition. Do NOT duplicate unchanged code into both branches of this clause (unless really really necessary, which is definitely not the case here). Don't duplicate `&inbytesleft` but rather move it to a new line.

Reply:
(oh, I just saw jralls proposed this himself - so don't take this too harsh, but before getting this merged, please make this basic code cleanup anyway)

Reply:
Absolutely, good point. Fixed.